submissions/madmaxWMFU/html-movie-booking/index.html

```diff
@@ -77,7 +77,7 @@ <h3 class="description__title">"Душа"</h3>
                                     </g>
                                 </g>
                             </svg>
-                            <span>12:00</span>
+                            <time>12:00</time>
```

Comment:
A good practice is providing tag `time` with attribute `datetime`, don't forget about him in future.